af/analysis/analysis/citizenlab_test_lists_updater.py

```diff
@@ -107,36 +108,59 @@ def create_citizenlab_cc_idx(conn):
         cur.execute(sql)
 
 
-@metrics.timer("rebuild_citizenlab_table_from_citizen_lab_lists")
-def rebuild_citizenlab_table_from_citizen_lab_lists(conf, conn):
+def rebuild_citizenlab_table_from_citizen_lab_lists(conf, citizenlab, conn):
     """Fetch lists from GitHub repository"""
-    ev = """INSERT INTO citizenlab (domain, url, cc, category_code, priority)
+    ev = """INSERT INTO citizenlab (domain, url, cc, category_code)
         VALUES %s"""
 
-    citizenlab = fetch_citizen_lab_lists()
-    compute_url_priorities(conn, citizenlab)
-    assert len(citizenlab) > 20000
-    assert len(citizenlab) < 1000000
-
     with conn.cursor() as cur:
-        log.info("Emptying citizenlab table")
+        log.info("Emptying PG citizenlab table")
         cur.execute("DELETE FROM citizenlab")
         log.info("Inserting %d citizenlab table entries", len(citizenlab))
         metrics.gauge("rowcount", len(citizenlab))
-        tpl = "(%(domain)s, %(url)s, %(cc)s, %(category_code)s, %(priority)s)"
+        tpl = "(%(domain)s, %(url)s, %(cc)s, %(category_code)s)"
         execute_values(cur, ev, citizenlab, template=tpl)
 
     if conf.dry_run:
         log.info("rollback")
         conn.rollback()
     else:
-        log.info("commit")
+        log.info("PG citizenlab commit")
         conn.commit()
-        log.info("citizenlab_table is ready")
+        log.info("PG citizenlab_table is ready")
+
+
+def query_c(click, query: str, qparams: dict):
+    click.execute(query, qparams, types_check=True)
+
+
+@metrics.timer("rebuild_citizenlab_table_from_citizen_lab_lists")
+def update_citizenlab_table_click(conf, citizenlab):
+    """Overwrite citizenlab_flip and swap tables atomically"""
+    if conf.dry_run:
+        return
+    click = Clickhouse("localhost")
+    log.info("Emptying Clickhouse citizenlab_flip table")
+    q = "TRUNCATE TABLE citizenlab_flip"
```

Comment:
How and where is the schema of the `citizenlab_flip` table defined?

Reply:
A copy of citizenlab. Both new and old tables are kept around to help debugging. They are very small.